### PR TITLE
MGDOBR-330: create kustomize overlay for stable environment

### PR DESCRIPTION
**Pull request by kostola**
[MGDOBR-330](https://issues.redhat.com/browse/MGDOBR-330)

### CODE REVIEW

**Review by r00ta — APPROVED** [review]

File: kustomize/overlays/stable/secrets/event-bridge-manager-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
...6H4nvS7sAEHUvni/UdN8vqyykQOZkOlpjeRRaGa4L4yb1RnByRB8OouX8ajUfhEFyLncGwouhiQ6F2vRdih7Swc93hB56jDouFADXOKRheyvZuHynFChF5Ral+7sCqIosoriZjW/DSYIwExUcsId6hhVQlc8UUXPMJZJ1V/9VkSTkos6TVPwYxXU/rrkjtRieXYTgtx1MrX6ePIahzGjKLQ9w7rQcEhnHUAxJ2vydx5GXbKM6/z4x/mRwNCjvOAvvYIcHSFmaKTKliQhyMvWiavko+EhlElJiCkp5pdYjLo9zFGrgQ5Y32hb0NKR9rSsFx6GEtICdinYdE2tx/nkw0Jxr/lfNAA5yWpz2esOAC0wZGI7G2LbCHj27Fi7tCBW/Q4R7iUFNEmZp81d4eBMZU+7sNIDFhcrfZQj57DB4uDIhIwZJWdDQDDVLZtcdYvdZ3VsUhWhQlEHd0MnwEVdS5lR9uF66ZDLMdOS0gwyr0+6XE9nesdisgcFML6lcbxLhEF1qMDxNq2a8xGtLi1xxmsLBhsQvdEBAfx2HtylKYcALQ1aqE/3R3aI2KrqNL1gWGmC19hTgHQSkMO2c3mGezgkGG51/mUMjw4H6RRMGK3E11SR0km5CFzLC2CTIIEXAgtssG4cYec9ll8EDAJ902XilxB1LcgLyzNESrWsFdU4be+BtnUofdWPLNqlJLE9mJIg2/78aPZh1cHVlXAUWg3Zm9lbRGW0+4QMWJZmxwE7YdYybKVPbKTSLQzQh9Z2JkOebhSHqQx2r+SXZ/00wTxpa65ABac072X6NfLqsv6Rct0eqcbrasbTa3KGTuqZnS9EL6tTxn3BGbNnAFxcIyAeO4YzhLp9v5ge+VGpOmrTYfsMZIbGY4SfQDHxMW6oikWapZt//Rozwgkpodg0gNo9XI94+Y01b6PI8xtE+w34vbwx1aujueNcgfv8EvyowR3xS9deZI0NNxHsMyD1Ds/Ka3n5CnfKNZ8sKIJQJcq9rF4jh3fM+0pURyBEJ92ohAbAhpDnoqaxB8hvCps1j4bkWhvcVbiqmSVm81XPg2pTcpnFcTjQQN4iC0WMf2zsKMM9pPajBAsEEKJg2/tSetTfZEbwy+rZKb/O1bGFNeV6XdccSPLr4TN06cpammWeiEsH8bqypy2GnXWZzCmlfanrinyBvzdLAfS/CnL/DHsLaA==
```
[the remark this answers is not in the document]
Comment (kostola) [reply kustomize/overlays/stable/secrets/event-bridge-manager-secrets.yaml]: Solved. Updated to use the token belonging to the newly created robot user.

File: kustomize/overlays/stable/secrets/keycloak-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  annotations:
+    sealedsecrets.bitnami.com/cluster-wide: "true"
+  creationTimestamp: null
+  name: keycloak-secrets
+spec:
+  encryptedData:
+    KEYCLOAK_PASSWORD: AgAI23aRUVsp7YZnoys7Ij9RGV0KZQOSR0YqG+GVFmbOQSgKPZyBnmuL3SBeTM3eVHqg/6QFsqZSVXQsgry/3q2uj+eX80zXlO5N7MQk1LaY1wadau/GgCN87TbK8UShilRa7YIq2Ow73V/93jaVAt4TJkYrz3UryVcwRdWzE8at3dbi4z3JFsbof8p9x2nLggSkFijHuqW73UfFk+vKtlu78Ek7QlzkgcqJkUK5IhDmdxKUx6uB5Zapim2MAJRsTWqVWbKmZnRj8LIhq1Bk6WLsn+U87pUP6kQjwuYTNObMaRUNeXNXqgT9faIw4aYOWXqyGSOUg9PoXfhm2mbddT4GXzxw/wqf6wOpjESaz2njDiJqfsW2k5LBTA8Y0XqSWfPdoFgC/5ahHavDXnIzOdE6fTxxBtRVYqZBaPVG0ywjj6UOqmUctfm6ZOBXkcFHwoT34XL/FJ8LbjLDn2JBgRZK4nv1kxXRuoP3w+3FzBtiu5EHIN7Emwi4mjZ5blI+p1bBhyGkOMswCQN1Obydifvw0j5lyU6Bl47/acRDpNtVErPt7bY/e5Ac/vXGbOBdyPKMn40/40ffffuNRykkveEcAVHluRZ15G3uKoqywaGRWIdSNU92VdmiTvH52BUHvvNujNzeqN/4Up4SbT7vld0BnthsqoQCWu9I8Vv88Rn14MWIWeQhjDRLfnodFOhYj0mc/pV1V9NK06uXAz1zkLM=
```
Comment (robpblake) [inline kustomize/overlays/stable/secrets/keycloak-secrets.yaml:10]: @kostola I assume the key used to encrypt these values is specific to our stable cluster?

**Review by robpblake — COMMENTED** [review]
@kostola Thanks. The only comment I have is about updating our `README.md` to make sure it's obvious to people that they need to use the correct key from each environment when encrypting secrets. 

cheers,
Rob

**Review by sutaakar — APPROVED** [review]

**Review by robpblake — DISMISSED** [review]
@kostola Please update the `kustomize/README.md` to reflect the names of the new environments and also to remind people to use the correct encryption key for each environment.

**Comment by kostola** [comment]
> @kostola Please update the `kustomize/README.md` to reflect the names of the new environments and also to remind people to use the correct encryption key for each environment.

@robpblake I will do this as part of [MGDOBR-331](https://issues.redhat.com/browse/MGDOBR-331) when the whole loop to update the stable cluster is well defined

**Pull request merged**